lib/logidze/model.rb

```diff
@@ -122,6 +135,14 @@ def apply_diff(version, diff)
       self
     end
 
+    def in_the_past?
```

Comment:
Let's prefix this method to avoid possible collisions, smth like `logidze_past?`.